update-bom/entrypoint.sh

```diff
@@ -44,23 +44,26 @@ fi
 
 if [ -f $CATALOG_FILE ]; then
     echo "Updating version catalog..."
-    if [ ! -z $bomProperty ]; 
+    if [ ! -z $bomProperty ];
     then
-        catalogProperty=$(echo $bomProperty | sed -r 's/([a-z0-9])([A-Z])/\1-\L\2/g' | sed -r 's/-version$//g')
+        if [[ -z "$catalogProperty" ]];
```

Comment:
@ilopmar with this change, the `gradle.properties` file of Micronaut ElasticSearch will need the following entry to be added:

```
catalogProperty=micronaut-elasticsearch
```